### PR TITLE
Add mobile menu

### PR DESCRIPTION
**Pull request by kazaneza**
>### In this pull request, I added a mobile menu

> Used javascript to for a dropdown menu

### CODE REVIEW

**Review by prolajumokeoni — COMMENTED** [review]
Hi @Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the javascript linters setup, and does not have a descriptive readme, please update the key features and future features and other sections yet to be updated in your readme

For this project both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters have to be setup
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by kazaneza** [comment]
Capture
@prolajumokeoni

File: index.html [hunk 1]
```diff
@@ -30,12 +22,19 @@
...
+          <button class="header__menu-btn">
+            <img src="images/Icon-Cancel.png" alt="Menu cancel button" />
+          </button>
+
+          <ul class="header__menu-content">
+            <li><a class="header__menu-link" href="#portfolio">Portfolio</a></li>
+            <li><a class="header__menu-link" href="#about">About</a></li>
+            <li><a class="header__menu-link" href="#contact">Contact</a></li>
+          </ul>
+        </div>
```
Comment (ShadyShawkat) [inline index.html:37]: - [ ] Please make sure that the mobile menu takes 100% of the screen's height to match the given design.

image

Comment (ShadyShawkat) [inline index.html:37]: - [ ] Kindly make sure that the user cannot scroll in the page itself when the mobile menu is open. Currently, if you have the mobile menu open, you can scroll past it onto the page content.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**